.github/workflows/jacoco-profile.yml

```diff
@@ -0,0 +1,51 @@
+name: JACOCO-PROFILE
+
+on:
+  push:
```

Comment:
Might as well add workflow_dispatch too?

Reply:
Good idea, updated it.

Reply:
Thank you!